service/kvmconfig/v2/resource/configmap/error.go

```diff
@@ -1,4 +1,4 @@
-package configmapv3
+package configmap
```

Comment:
The diff does not make sense here. Is it only the diff?

Reply:
Yes: https://github.com/giantswarm/kvm-operator/pull/354/files#diff-cafb67c12ad4101984064feedfe1c6a9